server/opts.go

```diff
@@ -2317,6 +2318,9 @@ func parseJetStream(v any, opts *Options, errors *[]error, warnings *[]error) er
 		for mk, mv := range vv {
 			tk, mv = unwrapValue(mv, &lt)
 			switch strings.ToLower(mk) {
+			case "strict":
+				opts.JetStreamStrict = true
```

Comment:
Better to do a type assertion here for `s, ok := vv.(bool)`, return an error if the type assert fails, and then set `JetStreamStrict` to the bool value. Otherwise setting `strict: false` in the config does the wrong thing.

Reply:
This resolved?

Reply:
Yep!